Client/Frontend/Settings/PlaylistSettingsViewController.swift

```diff
@@ -64,13 +64,6 @@ class PlaylistSettingsViewController: TableViewController {
         title = Strings.PlayList.playListSectionTitle
         
         dataSource.sections = [
-            Section(
-                rows: [
-                    .boolRow(title: Strings.PlayList.playlistToastShowSettingsOptionTitle,
-                             option: Preferences.Playlist.showToastForAdd),
-                ],
-                footer: .title(Strings.PlayList.playlistToastShowSettingsOptionFooterText)
-            ),
```

Comment:
Note to self: Remove these strings definitions (no longer will need to be translated)